tests/integration/targets/tanzu_mission_control_secret/aliases

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+disabled
```

Comment:
I would rather use
```suggestion
unsupported
```
to indicate that this works in general, but won't run in CI.